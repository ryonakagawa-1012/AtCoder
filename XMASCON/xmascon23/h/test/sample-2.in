20
0 130171465 825060067 310308502
32807742 472394518 556273189 402133431
18860036 731819479 614014131 516327409
540765157 764797868 805340356 885719657
