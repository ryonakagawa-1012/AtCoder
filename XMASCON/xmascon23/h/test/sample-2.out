220800069 663178740 402492745 787371697 340681055 764687638 248275291 239071271 579317784 142439206 202142764 449124008 17082226 208225793 292943419 63532736 780653882 855023292 155627784 988995756
